prow/getting_started_deploy.md

```diff
@@ -15,6 +15,8 @@ You need three things:
 2) a [personal access token][1] for the github bot user you want to act as prow
 
    - Prow uploads this token into a secret in your cluster
+   - The token must be stored in a text file
+   - The token must grant prow write access the GitHub repos. More details on GitHub ACLs are available [here][8]. 
```

Comment:
The "Manual Deployment" section [says](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#bot-account):

> The bot account used by prow must be granted owner level access to the Github orgs that prow will operate on.

Should this be changed to "write/push" access too or does it really need org level access?

Reply:
I cannot think of a use case where prow may need org level access, but I'm rather new to it, so there may be.

Reply:
@fejta Thanks! Should be changed to org level access then? Or maybe we should include this info the GitHub ACLs link?

Reply:
+1 we have noted this in both sections; maybe it's better to have a separate part about bot permissions be linked to in these sections. This purpose is not the same with scaling though.

Reply:
OK, so it seems to me that the permissions required depend on which functionality of prow it's used; so I think it would be good to document a minimum set of permissions required, and extra permissions required for particular commands.

What about a short section like:

The token must at least grant prow write/push access to the GitHub repositories. Prow can also be used to managed the GitHub organization, in which case org admin level permissions are required. For more details see [peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos), [branchprotector](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/branchprotector) and [here](https://github.com/kubernetes/test-infra/blob/master/prow/scaling.md#working-around-githubs-limited-acls).

This could be before both sections and we link to it from each of them.

Reply:
That paragraph is too long for the getting started guide (the tackle section)

We should provide the minimum amount of info necessary for them to get a minimally functional version of prow running (before they get overwhelmed and give up).

If they like this minimal version they can continue exploring what else they can make it do.